mbed_devices/mbed_tools.py

```diff
@@ -0,0 +1,4 @@
+"""Entry point for `mbed-tools` cli."""
```

Comment:
Should we add a link to the `mbed-tools` package here?

Reply:
Lovely idea.

Reply:
https://github.com/ARMmbed/mbed-devices/pull/52/commits/98647390808e76e657a6dedc4b4a5dc949ce9ce6